website/templates/website/index.html

```diff
@@ -165,15 +166,15 @@ <h3>Who can join?</h3>
     <p>Our group is entirely volunteer based, and our members come from various backgrounds and experiences. We are always looking for fresh ideas and new perspectives. We believe all our community members have something to offer as we design, develop, code, and implement projects.</p>
 
         <!-- Image carousel for group photos-->
-            <div id="aboutus-carousel" class="carousel slide" data-bs-ride="carousel">
+        <div id="aboutus-carousel" class="carousel slide" data-bs-ride="carousel">
 
             <!-- Indicators/dots -->
             <div class="carousel-indicators">
-                <button type="button" data-bs-target="#demo" data-bs-slide-to="0" class="active"></button>
-                <button type="button" data-bs-target="#demo" data-bs-slide-to="1"></button>
-                <button type="button" data-bs-target="#demo" data-bs-slide-to="2"></button>
-                <button type="button" data-bs-target="#demo" data-bs-slide-to="3"></button>
-                <button type="button" data-bs-target="#demo" data-bs-slide-to="4"></button>
+                <button id="indicator0" type="button" data-bs-target="#demo" data-bs-slide-to="0" class="active"></button>
+                <button id="indicator1" type="button" data-bs-target="#demo" data-bs-slide-to="1"></button>
+                <button id="indicator2" type="button" data-bs-target="#demo" data-bs-slide-to="2"></button>
+                <button id="indicator3" type="button" data-bs-target="#demo" data-bs-slide-to="3"></button>
+                <button id="indicator4" type="button" data-bs-target="#demo" data-bs-slide-to="4"></button>
```

Comment:
Bootstrap should handle changing the active slide using these buttons for us. The carousel has ID "aboutus-carousel", but the `data-bs-target` isn't using that ID. Changing from "#demo" to "#aboutus-carousel" should get it working and the new JS is no longer necessary.

Reply:
Well, that's cool but very frustrating to learn. I'll leave my script so that we can use it in the future, but I'll delete the code within the document's event listener